dashboard/app/models/programming_class.rb

```diff
@@ -55,20 +56,28 @@ def self.seed_all
 
   def self.seed_record(file_path)
     properties = properties_from_file(file_path, File.read(file_path))
```

Comment:
Not super familiar with our serializing process -- can you elaborate on why all the special casing for programming methods?

Reply:
Yeah! Methods are in a separate model so they can't be assigned directly, like `examples` can because it's just a json blob. Instead we need to iterate through the configs and find/create the ProgrammingMethod objects.

So the serialization will look something like:
```
{
  "category_key": null,
  "content": "",
  "examples": "[]",
  "external_documentation": "",
  "fields": "[]",
  "key": "painter",
  "name": "painter",
  "syntax": "",
  "tips": "",
  "methods": [
    {
      "name": "turnLeft()",
      "position": 0,
      "key": "turnleft",
      "content": null,
      "parameters": null,
      "examples": null,
      "syntax": null,
      "external_link": null
    },
    {
      "key": "moveforward",
      "position": 1,
      "name": "moveForward()",
      "content": null,
      "parameters": null,
      "examples": null,
      "syntax": null,
      "external_link": null
    }
  ]
}
```
with the methods being a sort of nested serialization of a different model.

Reply:
Got it, thanks!